Gordon360/Services/RecIMServices/LeagueService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.CCT.Context;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Gordon360.Services.RecIMServices
+{
+    public class LeagueService : ILeagueService
+    {
+        private readonly CCTContext _context;
+
+        public LeagueService(CCTContext context)
+        {
+            _context = context;
+        }
+
+
+        public IEnumerable<League> GetAllLeagues()
+        {
+            var leagues = _context.League.AsEnumerable();
+            return leagues;
+        }
+        public League GetLeagueByID(int leagueID)
+        {
+            return null;
+        }
```

Comment:
I know that this is just a temporary implementation, but in general please only return `null` on nullable types (e.g. you would update the return type of this method to be `League?` instead of `League`).

Also, the preferred way of handling temporary implementations in C# is to `throw new NotImplementedError()`

Reply:
I'll make sure I do that next time I just have placeholders :)